assets/dev/js/editor/elements/models/document.js

```diff
@@ -5,6 +5,6 @@ export default class Document extends BaseElementModel {
 		const childElType = childModel.get( 'elType' );
 
 		// Valid children.
-		return [ 'section', 'container' ].includes( childElType );
+		return [ 'section', 'container', 'a-container' ].includes( childElType );
```

Comment:
General question, is the name `a-container` still a good one? In the POC we referred to it as "Atomic Container", but now it is a Div Block".